src/transformState.js

```diff
@@ -4,8 +4,37 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    const current = actions[i];
```

Comment:
I suggest to use destructurization
```suggestion
    const { type, .... }= actions[i];
```